### DISCUSSION TITLE
Personal experience with the dashboard

### DISCUSSION THREAD

**Discussion by innerpeacez** [apache/skywalking#10695] [General]
I found that the current capabilities of the dashboard are relatively limited when using it. I am looking forward to more rich abilities of the dashboard，such as: 

1. coloring different values with different colors
2. dynamically changing units based on response data
3. multiple y-axes and units 
4. multiple types(bar, line, area ...) of charts in one widget
5. the table in the dashboard doesn't look very cool（It's just my subjective feeling, without any other implication）
6. the table have the option to hide column headers.

**Comment by wu-sheng** [comment]
We are not doing all, if you have use cases, we could discuss more.
After all, our UI is not for SkyWalking backend only.

**Comment by Superskyyy** [comment]
The boosterUI is rather new (from last year) and still has room for improvements. If you would like to enhance the UI by contributing new features, that would be awesome.
